### ISSUE TITLE
Canon Bootstrap

### ISSUE THREAD

**Issue by bradgignac** [rackerlabs/canon-react#19]
Rather than building React components for every Canon component, go with Canon Bootstrap and only build components that aren't already covered by react-bootstrap.

**Comment by bradgignac** [comment]
Chatted with @willcumbie in IRC for a while. To clarify, I don't really expect this repository to pull in react-bootstrap as a dependency. I was just using it as an example of time savings that you can get. Even if you all decide to re-implement everything, I would love to see it implemented with Canon Bootstrap markup and classes.

**Comment by adampickeral** [comment]
the problem is, there isn't really any kind of migration path defined for going from canon to canon-bootstrap, with no timelines on when that will be available. the idea with this repo is that if canon-bootstrap ever does replace canon, canon-react can serve as a migration path for teams using canon (and react), or at least serve as an example of how to migrate. so we won't be using canon bootstrap in this repo until it's readily consumable

**Comment by bradgignac** [comment]
Speaking in more concrete terms, who are you trying to help with this migration path? As far as I know, every team using React is already using Canon Bootstrap, and the UX org is backing Canon Bootstrap. Holding off on the transition will simply force other teams who are using React to write their own components.

**Comment by bradgignac** [comment]
@adampickeral Based on our conversation yesterday, what should the plan be for working on components that might eventually be replaced by react-bootstrap? What would your timeline be for considering a swap to canon-bootstrap?

**Comment by adampickeral** [comment]
@bradgignac I'm not too sure about timelines yet. I think what we'll do is after we deliver the next few components, we'll work on supporting both canon and canon-bootstrap.

**Comment by bradgignac** [comment]
Closing this issue while I wait on the more official communication from the Paper Street team.
